### DISCUSSION TITLE
Can not reorder connection

### DISCUSSION THREAD

**Discussion by orena1** [bonsai-rx/bonsai#867] [Q&A] [answered]
It seems that sometimes I can not reorder connection, e.g.:
![reorder_connection](https://user-images.githubusercontent.com/8983713/161832864-df78738b-b2c1-4eb4-abd4-7d5b193d38ae.gif)

Why is that? Is there something that I can do?
Thanks,

**Comment by glopesdev** [comment, marked as answer]
@orena1 this is a known issue: https://github.com/bonsai-rx/bonsai/issues/408

There are still some conditions under which the automatic visual layout algorithm is not correctly assigning the input order. It is on our backlog of issues to fix, and we keep working to figure out the remaining edge cases.

**Comment by orena1** [comment]
Got it, thanks
